diff --git a/plugins/scaffolder-backend-module-orchestrator/src/module.ts b/plugins/scaffolder-backend-module-orchestrator/src/module.ts
--- a/plugins/scaffolder-backend-module-orchestrator/src/module.ts
+++ b/plugins/scaffolder-backend-module-orchestrator/src/module.ts
@@ -17,7 +17,7 @@ import {
   coreServices,
   createBackendModule,
 } from '@backstage/backend-plugin-api';
-import { scaffolderActionsExtensionPoint } from '@backstage/plugin-scaffolder-node';
+import { scaffolderActionsExtensionPoint } from '@backstage/plugin-scaffolder-node/alpha';
 
 import {
   createGetWorkflowParamsAction,

